crates/ruff_python_parser/src/parser/mod.rs

```diff
@@ -1268,13 +1269,13 @@ impl RecoveryContextKind {
                     "Expected an expression or the end of the with item list".to_string(),
                 ),
             },
-            RecoveryContextKind::FStringElements(kind) => match kind {
-                FStringElementsKind::Regular => ParseErrorType::OtherError(
-                    "Expected an f-string element or the end of the f-string".to_string(),
+            RecoveryContextKind::FTStringElements(kind) => match kind {
+                FTStringElementsKind::Regular => ParseErrorType::OtherError(
+                    "Expected an f or t-string element or the end of the f or t-string".to_string(),
+                ),
+                FTStringElementsKind::FormatSpec => ParseErrorType::OtherError(
+                    "Expected an f or t-string element or a '}'".to_string(),
                 ),
```

Comment:
The standalone "f" looks a bit odd, maybe "f/t-string" or "f-string or t-string"?